build.sbt

```diff
@@ -51,6 +61,68 @@ lazy val units = crossProject(JVMPlatform, JSPlatform, NativePlatform)
         libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.5.0" % Test
     )
 
+// see also: https://github.com/lampepfl/dotty/issues/7647
+lazy val runtime = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+    .crossType(CrossType.Pure)
+    .in(file("runtime"))
+    .settings(name := "coulomb-runtime")
+    .dependsOn(
+        core % "compile->compile;test->test",
+        units % Test
+    )
+    .settings(
+        tlVersionIntroduced := Map("3" -> "0.8.0")
+    )
+    .settings(commonSettings: _*)
+    .settings(
+        // staging compiler is only supported on JVM
+        // but is also used to satisfy builds on JS and Native
+        libraryDependencies += "org.scala-lang" %% "scala3-staging" % scalaVersion.value
+    )
+    .platformsSettings(JSPlatform, NativePlatform)(
+        // any unit tests using staging must be excluded from JS and Native
+        Test / unmanagedSources / excludeFilter := HiddenFileFilter || "*stagingquantity.scala"
+    )
+
+// cats-parse doesn't seem to build for JS or Native
+lazy val parser = crossProject(JVMPlatform /*, JSPlatform, NativePlatform */ )
+    .crossType(CrossType.Pure)
+    .in(file("parser"))
+    .settings(name := "coulomb-parser")
+    .dependsOn(
+        core % "compile->compile;test->test",
+        runtime,
+        units % Test
+    )
+    .settings(
+        tlVersionIntroduced := Map("3" -> "0.8.0")
+    )
+    .settings(commonSettings: _*)
+    .settings(
+        libraryDependencies += "org.typelevel" %% "cats-parse" % "0.3.7"
```

Comment:
```suggestion
        libraryDependencies += "org.typelevel" %%% "cats-parse" % "0.3.7"
```

Reply:
doh.

Reply:
that works for JS, but still failing for Native

Reply:
```
[warn] 	Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading org.typelevel:cats-parse_native0.4_3:0.3.7
[error]   Not found
```

Reply:
@armanbilge   also having same issue with pureconfig, and `%%%` isn't fixing that

Reply:
- https://github.com/pureconfig/pureconfig/issues/1307